templates/internet-of-things/nvidia/contact-us.html

```diff
@@ -0,0 +1,135 @@
+{% extends "download/_base_download.html" %}
+
+{% block title %}Contact us{% endblock %}
+{% block meta_copydoc %}https://docs.google.com/document/d/1znVFFLSyTmDwCplvloGAlEI3p_dpGH8v82hbYzVfz0o/edit#{% endblock meta_copydoc %}
+
+{% block content %}
+
+
+<section class="p-strip--light">
+  <div class="row">
+    <div class="col-8">
+      <h1>Interest in optimised Ubuntu on NVIDIA Orin platforms?</h1>
+      <p>Canonical and NVIDIA announced a strategic collaboration to bring certified Ubuntu to the NVIDIA’s Orin platforms built with Tegra SoCs. The breadth of the collaboration spans the range of NVIDIA IoT platforms. Canonical and NVIDIA will partner on bringing Ubuntu to DRIVE Orin for in-vehicle embedded supercomputing, the IGX Orin for industrial- grade edge AI, Jetson AGX Orin for advanced robotics and other edge, Jetson NX Orin for lower-power budgets, and the smallest Jetson form factor with the Orin Nano.</p>
+      <p>With this partnership, Canonical and NVIDIA will deliver an optimised flavour of Canonical’s popular Ubuntu LTS to ecosystem partners and customers. The optimised kernel and images will integrate software features from NVIDIA with unparalleled security, reliability, and support for regulated enterprise  deployments.</p>
+      <p>Fill out the form to receive the latest update from us!</p>
+    </div>
+    <div class="col-4 u-vertically-center u-hide--small u-hide--medium">
+      {{
+				image (
+        url="https://assets.ubuntu.com/v1/cb6924a9-Nvidia_image_logo.svg",
+        alt="",
+        width="351",
+        height="259",
+        hi_def=True,
+        loading="auto"
+        ) | safe
+      }}
+    </div>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="row">
+    <div class="col-8">
+      <form action="/marketo/submit" method="post" onsubmit="buildCommentsForLead()" id="mktoForm_4953">
+        <fieldset class="u-no-margin--bottom">
+					<div class="js-formfield">
+            <h2 class="p-heading--3">Which NVIDIA Orin platforms are you interested in?</h2>
+            <label class="p-checkbox">
+              <input class="p-checkbox__input" type="checkbox" aria-labelledby="drive-orin" name="drive-orin" value="DRIVE Orin">
```

Comment:
```suggestion
              <input class="p-checkbox__input" type="checkbox" aria-labelledby="drive-orin" name="which-nvidia-orin-platforms" value="DRIVE Orin">
```
The name for all these checkboxes should be shared, it is what associates them to each other